modules/ipi-install-troubleshooting_unable-to-discover-new-bare-metal-hosts-using-the-bmc.adoc

```diff
@@ -0,0 +1,20 @@
+// This module is included in the following assemblies: 
+//
+// installing/installing_bare_metal_ipi/ipi-install-troubleshooting.adoc
+
+:_content-type: PROC
+[id="unable-to-discover-new-bare-metal-hosts-using-the-bmc_{context}"]
+= Unable to discover new bare metal hosts using the BMC
+
+In some cases, the installer will not be able to discover the new bare metal hosts and issue an error, because it cannot mount the remote virtual media share. 
+
+For example: 
+
+[source,json]
+----
+ProvisioningError 51s metal3-baremetal-controller Image provisioning failed: Deploy step deploy.deploy failed with BadRequestError: HTTP POST https://<bmc_address>/redfish/v1/Managers/iDRAC.Embedded.1/VirtualMedia/CD/Actions/VirtualMedia.InsertMedia returned code 400. Base.1.8.GeneralError: A general error has occurred. See ExtendedInfo for more information Extended information: [
+{"Message": "Unable to mount remote share https://<ironic_address>/redfish/boot-<uuid>.iso.", 'MessageArgs': ["https://<ironic_address>/redfish/boot-<uuid>.iso"], "MessageArgs@odata.count": 1, "MessageId": "IDRAC.2.5.RAC0720", "RelatedProperties": ["#/Image"], "RelatedProperties@odata.count": 1, "Resolution": "Retry the operation.", "Severity": "Informational"}
+].
+----
+
+In this situation, if you are using virtual media with an unknown certificate authority, you can configure your baseboard management controller (BMC) remote file share settings to trust an unknown certificate authority to avoid this error.
```

Comment:
We need to mention the procedure for Dell Machines, and this error so far is specific to Dell Hardware (there is a comment in the BZ  https://issues.redhat.com/browse/OCPBUGS-2630?focusedId=21641662&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-21641662

Reply:
I put a note in there. I have some bare metal servers running iDRAC 7, and the location is a little different, but the heading is the same. So that's why I mentioned remote file share settings. Also, the setting isn't yes|no in iDRAC7, so I'm trying to make it generic enough.

Reply:
I also put in a hyperlink from the firmware requirements to this issue.